esp/platform/application_config.cpp

```diff
@@ -151,6 +151,29 @@ bool addAuthNZSecurity(const char *name, IPropertyTree *legacyEsp, IPropertyTree
     return true;
 }
 
+bool addTestAuthSecurity(IPropertyTree *legacyEsp, IPropertyTree *appEsp, StringBuffer &bindAuth)
```

Comment:
@wangkx now that I look at it, you shouldn't need "testauth" specific routines in application_config.

Since the config is based on the SecurityManagers/SecurityManager format you should be able to make use of the "authNZ" mechanism.

Reply:
@afishbeck The "authNZ" mechanism works now.